sdk/translation/azure-ai-translation-text/samples/README.md

```diff
@@ -24,36 +24,103 @@ Translator Service is a cloud-based neural machine translation service that is p
 
 See the [README][README] of the Text Translator client library for more information, including useful links and instructions.
 
-## Common scenarios samples
+# Create Client
 
+Text Translation service is using two types of endpoints - Global and Custom. You can find more information in the [v3 Translator reference](TranslatorReference).
```

Comment:
Use () if you provide the link, use [] if you reference the link

```suggestion
Text Translation service is using two types of endpoints - Global and Custom. You can find more information in the [v3 Translator reference][TranslatorReference].
```

Reply:
Ah ok. Thank you!